pkg/jobrunaggregator/jobrunbigqueryloader/cmd.go

```diff
@@ -100,15 +100,54 @@ func (f *BigQueryTestRunUploadFlags) ToOptions(ctx context.Context) (*allJobsLoa
 
 	var jobRunTableInserter jobrunaggregatorlib.BigQueryInserter
 	var testRunTableInserter jobrunaggregatorlib.BigQueryInserter
+
+	var backendAlertTableInserter jobrunaggregatorlib.BigQueryInserter
+	var backendDisruptionTableInserter jobrunaggregatorlib.BigQueryInserter
+
 	if !f.DryRun {
 		ciDataSet := bigQueryClient.Dataset(f.DataCoordinates.DataSetID)
 		jobRunTable := ciDataSet.Table(jobrunaggregatorapi.LegacyJobRunTableName)
 		testRunTable := ciDataSet.Table(jobrunaggregatorlib.TestRunTableName)
 		jobRunTableInserter = jobRunTable.Inserter()
 		testRunTableInserter = testRunTable.Inserter()
+
+		// could start with dry run for the new uploaders if we wanted
+		// backendAlertTableInserter = jobrunaggregatorlib.NewDryRunInserter(os.Stdout, jobrunaggregatorapi.AlertsTableName)
+		// backendDisruptionTableInserter = jobrunaggregatorlib.NewDryRunInserter(os.Stdout, jobrunaggregatorapi.BackendDisruptionTableName)
+
+		// backendAlertTable := ciDataSet.Table(jobrunaggregatorapi.AlertsTableName)
+		// backendAlertTableInserter = backendAlertTable.Inserter()
+		// backendDisruptionTable := ciDataSet.Table(jobrunaggregatorapi.BackendDisruptionTableName)
+		// backendDisruptionTableInserter = backendDisruptionTable.Inserter()
+
 	} else {
 		jobRunTableInserter = jobrunaggregatorlib.NewDryRunInserter(os.Stdout, jobrunaggregatorapi.LegacyJobRunTableName)
 		testRunTableInserter = jobrunaggregatorlib.NewDryRunInserter(os.Stdout, jobrunaggregatorlib.TestRunTableName)
+
+		backendAlertTableInserter = jobrunaggregatorlib.NewDryRunInserter(os.Stdout, jobrunaggregatorapi.AlertsTableName)
+		backendDisruptionTableInserter = jobrunaggregatorlib.NewDryRunInserter(os.Stdout, jobrunaggregatorapi.BackendDisruptionTableName)
+	}
+
+	jobRunUploaderRegistry := JobRunUploaderRegistry{}
```

Comment:
I see multiple instances of the registry, what's the reason for that instead of having one like this https://github.com/openshift-eng/ci-test-mapping/blob/main/pkg/registry/registry.go#L187 ?

Reply:
Ah thanks, I didn't catch that they were different